src/CoursePageResources/index.scss

```diff
@@ -0,0 +1,48 @@
+.container-fluid {
```

Comment:
To agree with @xitij2000 's points below, we try to minimize semantic class names in our code to the bare minimum.  Instead, we can use Bootstrap's utility classes to accomplish most of these things.  

Paragon is built on top of Bootstrap, and provides a wealth of utility class names: https://edx.github.io/paragon/foundations/css-utilities

Also, as noted below, we definitely don't want to attach additional styling to existing Bootstrap class names; this will create very difficult to debug situations where unrelated code receive styling they're not expecting.

Reply:
Thank you. I will look into that.